tests/installation/change_desktop.pm

```diff
@@ -69,7 +69,7 @@ sub change_desktop {
     }
     else {
         if (!check_var('DESKTOP', 'gnome')) {
-            send_key_until_needlematch 'gnome-selected', 'down';
+            send_key_until_needlematch 'gnome-selected', 'down', 25;
```

Comment:
could you please rebase?, this change is already in the main branch, it is strange that it shows like this...

Reply:
Its not strange, I changed it yesterday after the VR failed because of too little 'key-downs'. Didn't see that this was in the main branch already. Well, rebase done, now it should be complete, didn't find any active schedules that can be transformed more to libyui.